### PR TITLE
Assign Order API

### PR DESCRIPTION
**Pull request by ssagheer532**
This function updates the assignToDriver field for the order specified.

### CODE REVIEW

File: functions/assignOrder/config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "name": "assignOrder",
+    "private": false,
+    "run_as_system": true
```
Comment (kwilsonO) [inline functions/assignOrder/config.json]: for the functions which need to use the context to get user data I think it needs to not run as system

File: functions/assignOrder/source.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+exports('5e76aad15f06170a32d8f8b5', '2')
+
+*/
+
+exports = function(orderId, driverId){
+
+  let db = context.services.get("mongodb-atlas").db("stayneighbor")
+  let collection = db.collection("orders")
+  let query = {_id: BSON.ObjectId(orderId)}
+  let updateCmd = {$set: {assignedToDriver: driverId}}
```
Comment (kwilsonO) [inline functions/assignOrder/source.js]: can we also set the assignedToOrgId, you will be able to get it from:

context.user.custom_data.organizationId

Comment (ssagheer532) [reply functions/assignOrder/source.js]: Good idea. I made the change but am not really sure how to test it. I'd need to create a user on the ui right?

File: functions/assignOrder/source.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+exports('5e76aad15f06170a32d8f8b5', '2')
+
+*/
+
+exports = function(orderId, driverId){
+
+  let db = context.services.get("mongodb-atlas").db("stayneighbor")
+  let collection = db.collection("orders")
+  let query = {_id: BSON.ObjectId(orderId)}
+  let updateCmd = {$set: {assignedToDriver: driverId, assignedToOrg: context.user.custom_data.organizationId}}
```
Comment (ssagheer532) [inline functions/assignOrder/source.js:15]: This line may be wrong. I'm not sure how to test if `context.user.custom_data.organizationId` will give me the correct value. @kwilsonO @dillonharless18 do either of you have an idea?

Comment (kwilsonO) [reply functions/assignOrder/source.js:15]: this looks correct to me!

**Review by kwilsonO — APPROVED** [review]

**Pull request merged**